core/camel-support/src/main/java/org/apache/camel/support/PropertyBindingSupport.java

```diff
@@ -258,11 +257,25 @@ private static boolean doAutowireSingletonPropertiesFromRegistry(CamelContext ca
      * @return              true if one or more properties was bound
      */
     public static boolean bindProperties(CamelContext camelContext, Object target, Map<String, Object> properties) {
-        return bindProperties(camelContext, target, properties, true, true, true, true, true);
+        return bindProperties(camelContext, target, properties, null);
     }
 
     /**
-     * Binds the properties to the target object, and removes the property that was bound from properties.
+     * Binds the properties with the given prefix to the target object, and removes the property that was bound from properties.
+     * Note that the prefix is removed from the key before the property is bound.
+     *
+     * @param camelContext  the camel context
+     * @param target        the target object
+     * @param properties    the properties where the bound properties will be removed from
+     * @param optionPrefix  the prefix used to filter properties
```

Comment:
Also add the optionPrefix to the Builder (see top of class) so you can use this style as well